script/chart_sync_utils.sh

```diff
@@ -58,6 +58,13 @@ updateRepo() {
     rm "${targetChartPath}/Chart.yaml.bk"
     # DANGER: This replaces any tag marked as latest in the values.yaml
     sed -i.bk 's/tag: latest/tag: '"${targetTag}"'/g' "${targetChartPath}/values.yaml"
+    # Replace quay.io images for docker.io
+    sed -i.bk 's/registry: quay.io/registry: docker.io/g' "${targetChartPath}/values.yaml"
+    # Remove v prefix from tags
+    sed -i.bk 's/tag: v\(.*\)/tag: \1/g' "${targetChartPath}/values.yaml"
+    # Use bitnami images
+    sed -i.bk 's/repository: helmpack\/\(.*\)/repository: bitnami\/kubeapps-\1/g' "${targetChartPath}/values.yaml"
```

Comment:
we could combine these with `...repository: (helmpack|kubeapps)\/...`, but not that important as again, we will probably just apply this chart-wide once we have the bitnami version of keycloak-gatekeeper.